index.html

```diff
@@ -4,6 +4,7 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
-
+<h1>Michael Silverberg</h1>
```

Comment:
Please indent your tags.
Refer to this guide for indentation rules:
https://google.github.io/styleguide/htmlcssguide.xml

Change to the following:

```html
<body>
  <h1>Michael Silverberg</h1>
  <img src="http://i.giphy.com/XjZrD0wU10FX2.gif" alt="flying goat">
</body>